app/src/components/Day.js

```diff
@@ -5,13 +5,17 @@ import Button from "react-bootstrap/Button";
 import Card from "react-bootstrap/Card";
 import ListGroup from "react-bootstrap/ListGroup";
 import ListGroupItem from "react-bootstrap/ListGroupItem";
+import DatePicker from "react-datepicker";
+import { Link, useParams, useHistory } from "react-router-dom";
 
 import { moonPhases } from "../MoonPhases";
 import * as apiClient from "../apiClient";
 //rename data to props
 const Day = ({ initialData }) => {
-  const today = dayjs();
-  const [date, setDate] = React.useState(today);
+  let { day } = useParams();
```

Comment:
does this useParams() take out the needs for 
` const today = dayjs();`

I see that you still have that imported as a library the dayJS